test/integration/log_system.cc

```diff
@@ -698,9 +698,7 @@ TEST_F(LogSystemTest, RecordAndPlayback)
   // Create temp directory to store log
   this->CreateLogsDir();
 
-  // Used to count the expected number of poses recorded. Counting is necessary
```

Comment:
Is this no longer true?

Reply:
This is no longer true. From my understanding, `dynamic_pose/info` was being published at [60Hz from `SceneBroadcaster`](https://github.com/ignitionrobotics/ign-gazebo/blob/657ca6dca0d7b1200c284c10a5776fb4f6aad3df/src/systems/scene_broadcaster/SceneBroadcaster.cc#L554-L555) & the number of expected poses varied based on `ign-transport`'s throttle mechanism, which was why they were counted.

Now, the `changed_state` topic is being published (almost) every time `LogRecord::PostUpdate` is called:
https://github.com/ignitionrobotics/ign-gazebo/blob/a3438b62b7f9643ce33ae96e1540b4eb02dfb82d/src/systems/log/LogRecord.cc#L702-L704

This test in particular (`LogSystemTest.RecordAndPlayback`) is using a double pendulum model, which always has new poses (i.e., states) for joints, so for 1000 iterations we are expecting 1000 state messages:
https://github.com/ignitionrobotics/ign-gazebo/blob/a3438b62b7f9643ce33ae96e1540b4eb02dfb82d/test/integration/log_system.cc#L840

Reply:
Thanks for the explanation, Jenn!. Sounds good to me.